### PR TITLE
git embeddings feature xpassing tests for go and ruby

### PR DESCRIPTION
**Pull request by rachelyangdog**
## Motivation
xpassing tests for git embeddings feature for Go and ruby

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: tests/parametric/test_tracer.py [hunk 1]
```diff
@@ -134,8 +134,8 @@ def test_tracer_commit_sha_environment_variable(
             },
         ],
     )
-    @missing_feature(context.library == "golang", reason="golang does not strip credentials yet")
-    @missing_feature(context.library == "nodejs", reason="nodejs does not strip credentials yet")
+    # @missing_feature(context.library == "golang", reason="golang does not strip credentials yet") # causing easy wins
```
Comment (Kyle-Verhoog) [inline tests/parametric/test_tracer.py:137]:
```suggestion
```

File: tests/parametric/test_tracer.py [hunk 2]
```diff
@@ -134,8 +134,8 @@ def test_tracer_commit_sha_environment_variable(
...
+    @missing_feature(context.library == "nodejs", reason="nodejs does not strip credentials yet") # causing easy wins
```
Comment (Kyle-Verhoog) [inline tests/parametric/test_tracer.py:138]: not really sure what "causing easy wins" means 🤔

File: tests/parametric/test_tracer.py (hunk 1 above)
Comment (Kyle-Verhoog) [reply tests/parametric/test_tracer.py:137]: should we just remove it altogether? I don't see any value in keeping it. Do we have to update the manifest/go.yaml file with the version that support was added?

Comment (rachelyangdog) [reply tests/parametric/test_tracer.py]: Deleted the comment and also updated the manifest file! thank you

File: manifests/golang.yml [hunk 3]
```diff
@@ -458,8 +458,6 @@ tests/:
       Test_Trace_Sampling_Resource: v1.60.0
       Test_Trace_Sampling_Tags: v1.60.0
       Test_Trace_Sampling_Tags_Feb2024_Revision: v1.64.0-dev
-    test_tracer.py:
-      Test_TracerSCITagging: v1.48.0
```
Comment (Kyle-Verhoog) [inline manifests/golang.yml:462]: we still need this so that if we run the tests against an older version they pass.

File: manifests/ruby.yml [hunk 4]
```diff
@@ -355,8 +355,6 @@ tests/:
       Test_Trace_Sampling_Tags: missing_feature
       Test_Trace_Sampling_Tags_Feb2024_Revision: missing_feature
       Test_Trace_Sampling_With_W3C: missing_feature
-    test_tracer.py:
-      Test_TracerSCITagging: missing_feature
```
Comment (Kyle-Verhoog) [inline manifests/ruby.yml:359]: can we find the version that this was implemented in? (same motivation as having the go version)

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**